src/test/resources/requests/authRequest200_single.json

```diff
@@ -0,0 +1,5 @@
+{
+  "date":"2024-02-08",
+  "authType": "UKNW",
+  "eoris" : ["GB0000000200"]
```

Comment:
Please change to GB000000000200
GB0000000200 does not match the regex for EORis

Reply:
Changed EORI